src/sentry/integrations/vercel/webhook.py

```diff
@@ -1,24 +1,139 @@
+import hashlib
```

Comment:
This file has not been changed and constitutes most of the diff.

`webhook.py` is our convention for webhook files, so I renamed `generic_webhook.py` into `webhook.py` and delete the previous, outdated version of `webhook.py`.